docs/HSL-mallitoiden_tilaajan_ohje.md

```diff
@@ -1,3 +1,7 @@
+---
+sort: 7
+---
```

Comment:
Like this.

Reply:
Great! Is my order suggestion above ok? If so, could you edit accordingly?

Reply:
Well, I have tried to sort them like this:

1. General info and instructions about the model system
2. More specifically about the data and methods HSL currently provides (the model system can be used with other data as well)
3. Instructions when using the model system in HSL projects (only a small subset of all helmet projects)
4. Examples of specific output (includes only transit assignmnent examples at the moment)

I think my order makes more sense.